test/run

```diff
@@ -153,8 +175,49 @@ def add_e2e_tests(litani, test_dir, counter, output_dir, fast):
                 ci_stage="test",
                 description=f"{test_file.stem}: transform jobs",
                 inputs=add_jobs_output,
-                phony_outputs=run_build_input,
+                phony_outputs=transform_jobs_output,
                 cwd=run_dir)
+            optional_outputs.append(transform_jobs_output)
+
+        if add_get_jobs:
+            get_jobs_output = str(uuid.uuid4())
+            litani_add(
+                litani, counter,
+                command=collapse(f"""
+                    {e2e_test_dir / 'run'}
+                    --test-file {test_file}
+                    --litani {litani}
+                    --run-dir {run_dir}
+                    --operation get-jobs"""),
+                pipeline=f"End-to-end: {test_file.stem}",
+                ci_stage="test",
+                description=f"{test_file.stem}: get jobs",
+                inputs=add_jobs_output,
+                phony_outputs=get_jobs_output,
+                cwd=run_dir)
+            optional_outputs.append(get_jobs_output)
+
+        if add_set_jobs:
+            set_jobs_output = str(uuid.uuid4())
+            litani_add(
+                litani, counter,
+                command=collapse(f"""
+                    {e2e_test_dir / 'run'}
+                    --test-file {test_file}
+                    --litani {litani}
+                    --run-dir {run_dir}
+                    --operation set-jobs"""),
+                pipeline=f"End-to-end: {test_file.stem}",
+                ci_stage="test",
+                description=f"{test_file.stem}: set jobs",
+                inputs=add_jobs_output,
```

Comment:
Note that `get_jobs` and `set_jobs` are not currently ordered with respect to each other. They both depend on the same file, so they're going to potentially run at the same time.

Reply:
Is that an issue? I thought that was how it is supposed to work